### PR TITLE
feat: add 64bits support

### PR DESCRIPTION
**Pull request by immortal-tofu**
- Tests are running on the CI launched manually with latest version of KMS/fhevm-go https://github.com/zama-ai/zbc-ethermint/actions/runs/7844274084/job/21408137996
- Mocks has been updated to support 64bits
- Contracts has been moved to euint64 (ERC20/Governor/Comp/Identity/...)
- fhevmjs has a prerelease which supports 64bits https://github.com/zama-ai/fhevmjs/releases

### CODE REVIEW

**Comment by immortal-tofu** [comment]
Tests won't work until we have a proper dev image cc @leventdem

**Comment by jatZama** [comment]
> Tests won't work until we have a proper dev image cc @leventdem

is this what is causing the nonce errors in the tests?

File: codegen/overloadTests.ts [hunk 1]
```diff
@@ -50,7 +60,12 @@ export const overloadTests: { [methodName: string]: OverloadTest[] } = {
     { inputs: [0x00000001], output: 0xffffffff },
     { inputs: [0x00000002], output: 0xfffffffe },
   ],
-  not_euint32: [{ inputs: [0x0000fffe], output: 0xffff0001 }],
```
Comment (jatZama) [inline codegen/overloadTests.ts:53]: not sure to understand this test output?

**Review by jatZama — COMMENTED** [review]
correct not_euint32 in overload.ts

File: codegen/overloadTests.ts (hunk 1 above)
Comment (immortal-tofu) [reply codegen/overloadTests.ts:53]: This test didn't change in the last commit. You are probably looking at an old one :)

**Comment by immortal-tofu** [comment]
> > Tests won't work until we have a proper dev image cc @leventdem
> 
> is this what is causing the nonce errors in the tests?

Yes, basically the first test to fail make the next tests failing for a nonce reason. Two things change regarding running tests:
- We need the new stack with 64bits support and tfhe-rs 0.5.1
- We need a computer capable to run multiplication over two euint64 in less than 7seconds: this is not possible on my M1 or on the default CI. That's why I switched to `large_ubuntu_16` in GitHub Actions.

**Review by david-zk — APPROVED** [review]
💪

File: codegen/overloadTests.ts (hunk 1 above)
Comment (david-zk) [reply codegen/overloadTests.ts:53]: @jatZama that is binary not, bit inversion, not boolean not

Comment (jatZama) [reply codegen/overloadTests.ts:53]: I know this, here I was replying to a previous old commit version which has been corrected in a following commit.

**Review by jatZama — APPROVED** [review]

**Pull request merged**